src/test/ui/async-await/issue-93197.rs

```diff
@@ -0,0 +1,15 @@
+// Regression test for #93197
+// build-pass
```

Comment:
Check pass would be sufficient, but OK :)

Reply:
Good point, I changed it.

My default for these is build-pass because then we generate MIR and make sure typeck and MIR agree with each other.